docs/build/program/apis/robot.md

```diff
@@ -882,7 +882,7 @@ For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/r
 
 ```go
 metadata := machine.GetCloudMetadata()
-fmt.Println(metadata.RobotPartID)
+fmt.Println(metadata.MachinePartID)
 fmt.Println(metadata.PrimaryOrgID)
 fmt.Println(metadata.LocationID)
```

Comment:
While we're here let's change it to this instead because we don't print output anywhere else but I think it's useful to show the three components
```suggestion
machine_part_id = metadata.MachinePartID
primary_org_id = metadata.PrimaryOrgID
location_id = metadata.LocationID
```